pbs/pbsrequest.go

```diff
@@ -127,6 +135,17 @@ func ParsePBSRequest(r *http.Request, cache cache.Cache) (*PBSRequest, error) {
 	if pbsReq.Device == nil {
 		pbsReq.Device = &openrtb.Device{}
 	}
+	if pbsReq.SDK == nil {
+		pbsReq.SDK = &SDK{}
+	}
+	if pbsReq.SDK.Version != "0.0.1" {
+		if pbsReq.PBSUser != nil {
```

Comment:
What happens if SDK version == 0.0.1? Should it change the gender to string and then forward to bidders?